### PR TITLE
refactor(content-types): update research CT - INNO-2010

### PR DESCRIPTION
**Pull request by kalinchernev**
# PR description

Nothing special to note.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/systems/ec/implementations/react/templates/pages/research-area-spoke/src/ResearchAreaSpokePage.jsx [hunk 1]
```diff
@@ -381,11 +387,184 @@ const ResearchAreaSpokePage = ({ siteHeader, footer, template }) => (
...
+
               <p className="ecl-u-type-paragraph ecl-u-type-color-black ecl-u-mt-m ecl-u-mb-none">
-                <Link href="/example" label="More information" /> about Atlantic
-                Ocean sea basin strategies.
+                BONUS was the joint Baltic Sea research and development
+                programme from 2010-2017.
               </p>
 
+              <UnorderedList
+                className="ecl-u-mt-m ecl-u-mb-none"
```
Comment (papegaill) [inline src/systems/ec/implementations/react/templates/pages/research-area-spoke/src/ResearchAreaSpokePage.jsx]: should be `ecl-u-mt-xs`

**Review by papegaill — CHANGES_REQUESTED** [review]
@kalinchernev Only a small spacing remark for the desktop version. But all the spacing defined with mobile mockup are not set. for example:

- `3xl` top margin for `<h2>` becomes `xl` for mobile.
- `l` top margin for `<h3>` becomes `m` and so on. 

Do you want me to update the PR regarding this?

**Comment by kalinchernev** [comment]
Thanks @papegaill updates applied.

**Review by papegaill — APPROVED** [review]
@kalinchernev Thank you for the updates, approving the PR, but we need to clarify that spacing rules are to be applied to `lg` breakpoint. We don't have specification for the spacings between mobile and desktop.. I would find it more logical to apply the rules from the breakpoint `sm` or`md` instead of `lg`, @emeryro, @planctus ?

**Comment by papegaill** [comment]
@kalinchernev seems there is some drone issue to be fixed before merging.

**Pull request merged**